Update experiment-rollout.js

Lordcobcob opened the pull request:
Added new tag (hopefully) for automod on user profile (guild rollout)

Lordcobcob committed:
Update experiment-rollout.js

Added new tag (hopefully) for automod on user profile (guild rollout)

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -104,6 +104,10 @@ const data = {
             }
         ]
     },
+    
+
+
```
```suggestion
```
what the heck are these?

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -124,6 +128,31 @@ const data = {
             }
         ]
+    },
+     automod_on_user_profile: {
+
+        rate: 30,
+
+        experimentType: 0,
+
+        rolloutType: 0,
+
+        requirements: [
+
+            {
+
+                type: 1,
+
+                value: ['COMMUNITY'],
+
+                rate: 30,
+
+                ranges: [[3000, 6000]]
+
+            }
+
+        ]
+
```
```suggestion
     automod_profile: {
        rate: 30,
        ranges: [[3000, 6000]]
        experimentType: 0,
        rolloutType: 0,
        requirements: [
            {
                type: 1,
                value: ['AUTOMOD_TRIGGER_USER_PROFILE'],
                rate: 100,
                ranges: [[0, 10000]]
            },
            {
                type: 1,
                value: ['COMMUNITY'],
                rate: 30,
                ranges: [[3000, 6000]]
            }
        ]
```

Lordcobcob committed:
Update experiment-rollout.js

Uh tolg said to do this 👌

Co-authored-by: ✨Tolgchu✨ <tolgchu-studios@outlook.com>

Lordcobcob committed:
Update experiment-rollout.js

Also smthn tolg told me to do I'm trying to do this on mobile my man it's hard 😭😭

Co-authored-by: ✨Tolgchu✨ <tolgchu-studios@outlook.com>

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -104,6 +104,7 @@ const data = {
             }
         ]
     },
+    },
```
```suggestion
```
i missed that

Lordcobcob committed:
Update experiment-rollout.js

Oops double text fixed

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -104,6 +104,7 @@ const data = {
             }
         ]
     },
+
```
```suggestion
```
still have space 🤨

Lordcobcob committed:
Update experiment-rollout.js

I GET IT MOBILE GITHUB EDITING IS WEIRD ASF STOP BULLYING Me 😭😭😭😭😭😭

Co-authored-by: ✨Tolgchu✨ <tolgchu-studios@outlook.com>

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -124,6 +124,26 @@ const data = {
+    },
+     automod_profile: {
+        rate: 30,
+        ranges: [[3000, 6000]]
```
```suggestion
        ranges: [[3000, 6000]],
```
missing comma

Tolga1452 reviewed (changes requested):
ig that's the latest one

Lordcobcob committed:
Update experiment-rollout.js

Sorry 😔 lmao

Co-authored-by: ✨Tolgchu✨ <tolgchu-studios@outlook.com>
